src/app/character-sheet/character-sheet.component.html

```diff
@@ -21,7 +21,8 @@
     <header>Defense</header>
 
     <div><b>AC</b> {{char.ac + (char.abilityScores.dex | abilityMod)}}</div>
-    <div><b>HP: </b> <input type="number" [(ngModel)]="char.currentHp" (ngModelChange)="updateCharacter({currentHp: char.currentHp})"/> / {{char.hp}}</div>
+    <div><b>HP: </b> <input type="number" [(ngModel)]="char.currentHp" (ngModelChange)="updateCharacter({currentHp: char.currentHp})"/> {{char.currentHp}} / {{char.hp}} <button>Damage</button>
```

Comment:
I think we need to save these values on the rawCharacter.
A good way to think about it is anything that needs to be saved, should be on the raw character.